README.md

```diff
@@ -59,6 +67,8 @@ to those three directories), and simply type
 
 and the mlboard headers are found in `/usr/local/include/mlpack/`.
 
+To run
```

Comment:
Why have you written this? I think the last sentenc is quite complete.

Reply:
To run test, you can issue the following command:

```
    $ ./mlboard_tests
```

It was half cooked